lib/actions/options/read-write.ts

```diff
@@ -38,23 +38,19 @@ export class ContractsOptionsReadWriteActions
     option: Option<Underlying, Collateral>,
     amount: MonetaryAmount<Collateral>,
     btcAddress: BtcAddress
-  ) {
+  ): Promise<void> {
     const pair = await this.contracts.getPair(option.address);
     const premium = await this.estimatePremium(option, amount);
-    await pair.write(premium.toString(), amount.toString(), btcAddress);
+    pair.write(premium.toString(), amount.toString(), btcAddress);
```

Comment:
We either want to await or return this.

Reply:
This returns a `ConfirmationNotifier` which cannot be awaited, the `await` was a noop. It could be returned but when I briefly tried it generated a _ton_ of issues with typing so I left it this way for now.

Reply:
Oh, that was not returning a promise in the first place. My bad, I thought it was.